pkg/nimble/Makefile

```diff
@@ -14,6 +14,9 @@ ifeq (llvm,$(TOOLCHAIN))
 # `nimble/controller/src/ble_ll_adv.c` isn't used in our compilation path, so
 # tell LLVM/clang not to be so pedantic with this.
   CFLAGS += -Wno-unused-function
+# Workaround, until #566 in NimBLE is merged and the NimBLE version in RIOT is
```

Comment:
Could you provide this as link please? It makes it much quicker to check if the first condition is met ;-).